docs/release-notes/0.2/release-0.2.md

```diff
@@ -0,0 +1,61 @@
+# ML.NET 0.2 Release Notes
+
+We would like to thank the community for the engagement so far and helping us shape ML.NET.
+
+Today we are releasing ML.NET 0.2. This release focuses on addressing questions/issues, adding clustering to the list of supported machine learning tasks, enabling using data from memory to train models, easier model validation, and more.
+
+### Installation
+
+Supported platforms: Windows, MacOS, Linux (see [supported OS versions of .NET Core 2.0](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) for more detail)
```

Comment:
>Supported platforms: Windows, MacOS, Linux [](start = 0, length = 42)

I am somewhat hostile to the idea of using language in this fashion, especially when there is so little cost to actually using proper grammatic sentences. So instead of "Supported platforms: Windows, MacOS, Linux" we may use "Supported platforms **are** Windows, MacOS, **and** Linux." You could even use the active voice *we support* as opposed to "supported" for bonus points.

Reply:
Addressed.